hello-cdep/README.md

```diff
@@ -0,0 +1,63 @@
+Hello-libs
+=========
+hello-cdep is an Android sample that demos [CDEP](https://github.com/google/cdep) native lib distribution system usage.
```

Comment:
Title says "Hello-libs" but description says "Hello-cdep". Seems like the latter is better.